### PR TITLE
Disable fast math for gl_Position

### PR DESCRIPTION
**Pull request by trenouf**
I accidentally disabled the existing fix to disable fast math for
gl_Position in my change "Switched LowerGlobal to use builder methods
for input/output". This commit re-enables it.

Change-Id: I175e8bb5ffe0611fecbe2a85a104b526d9cf7d80

### CODE REVIEW

File: lower/llpcSpirvLowerAlgebraTransform.cpp [hunk 1]
```diff
@@ -411,16 +411,25 @@ void SpirvLowerAlgebraTransform::visitCallInst(
     }
     else
     {
-        // Disable fast math for gl_Position
+        // Disable fast math for gl_Position.
+        // TODO: Having this here is not good, as it requires us to know implementation details of Builder.
+        // We need to find a neater way to do it.
         auto calleeName = pCallee->getName();
+        uint32_t builtIn = InvalidValue;
+        Value* pValueWritten = nullptr;
```
Comment (amdrexu) [inline lower/llpcSpirvLowerAlgebraTransform.cpp]: pValueWritten  -> pValueToWrite

File: lower/llpcSpirvLowerAlgebraTransform.cpp [hunk 2]
```diff
@@ -411,16 +411,25 @@ void SpirvLowerAlgebraTransform::visitCallInst(
...
-                DisableFastMath(pOutput);
-            }
+            builtIn = cast<ConstantInt>(callInst.getOperand(0))->getZExtValue();
+            pValueWritten = callInst.getOperand(callInst.getNumArgOperands() - 1);
+        }
+        else if (calleeName.startswith("llpc.call.write.builtin"))
+        {
+            builtIn = cast<ConstantInt>(callInst.getOperand(1))->getZExtValue();
+            pValueWritten = callInst.getOperand(0);
+        }
```
Comment (amdrexu) [inline lower/llpcSpirvLowerAlgebraTransform.cpp]: Add a blank line after this line

**Comment by trenouf** [comment]
That test failure seems spurious.

**Comment by trenouf** [comment]
I will land this.

**Pull request merged**